packages/v4/patternfly-docs/content/design-guidelines/demos/primary-detail/primary-detail.md

```diff
@@ -27,39 +30,40 @@ When the page content has a white background, the drawer can be placed on the sa
 - Input forms
```

Comment:
ANd same here, link to the components listed under "common detail components"

Reply:
I added the links to the items on the list except for the markdown text form because I wasn't sure where it was on PF.